### PR TITLE
docs(api): merge slots and scope-slots

### PR DESCRIPTION
**Pull request by MajesticPotatoe**
## Description
Merges scope-slots with slots into slots.

## Motivation and Context
Because slots.

## How Has This Been Tested?
visually | docs

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/docs/src/lang/en/components/Alerts.json [hunk 1]
```diff
@@ -1,78 +1,76 @@
...
-  },
-  "scopedSlots": {
-    "close": "Slot for icon used in **dismissible** prop."
-  },
-  "functions": {
-    "toggle": "Toggles the alert's active state. Available in the close slot and used as the click action in **dismissible**."
-  }
-}
+{
+  "header": "# Alerts",
```
Comment (KaelWD) [inline packages/docs/src/lang/en/components/Alerts.json]: What changed here?

Comment (MajesticPotatoe) [reply packages/docs/src/lang/en/components/Alerts.json]: Slot and scopeslots merged. Not sure why git decided that everything changed.

Comment (KaelWD) [reply packages/docs/src/lang/en/components/Alerts.json]: Did you mess with line endings?

Comment (MajesticPotatoe) [reply packages/docs/src/lang/en/components/Alerts.json]: Don't think so, unless linter or something changed them or windows is being cranky. My git config is set to leave them as is though.

Comment (KaelWD) [reply packages/docs/src/lang/en/components/Alerts.json]: Nah boi you changed that shit to crlf

**Pull request merged**